src/components/itemHelper.js

```diff
@@ -331,6 +331,30 @@ export function supportsMediaSourceSelection (item) {
     return true;
 }
 
+export function sortTracks (trackA, trackB) {
+    if (trackA.IsExternal && trackB.IsExternal) {
```

Comment:
_visually - not tested_
```suggestion
    if (trackA.IsExternal === trackB.IsExternal) {
```
Otherwise, if they are both non-external, `-1` will be returned instead of checking the other properties.

The same for others.